### DISCUSSION TITLE
How to start trustwallet core project

### DISCUSSION THREAD

**Discussion by 0x00trust** [trustwallet/wallet-core#1573] [Q&A]
Hello
I'm new user
How to start build trustwallet core to new wallet?
like as 

coinpaprika
IFWallet
Alice
Crypto.com
Frontier
Belco
Pumapay

Thank you

**Comment by hewigovens** [comment]
https://developer.trustwallet.com/wallet-core/integration-guide

**Reply by 0x00trust** [reply]
Thank you for your info.
Can you make video to get started.
By the way
May i have questions
- Trustwallet is de-Fi? and have many server?
- If all server trustwallet shutdown. Where is my money?
- If other wallet used trustwallet-core like as IFWallet ,Crypto.... Can login with Recovery Phrase register with trustwallet? I'm not try yet .
- Can make other wallet used trustwallet-core install on any computer can sysn data so any one can login with Recovery Phrase to restore money.?

**Reply by optout21** [reply]
- Trust Wallet is decentralized, with multiple redundant nodes for popular chains
- Trust Wallet is non-custodial, funds are always stored with the decentralized networks (Ethereum, bitcoin, BSC, ...)
- Trust Wallet uses BIP39 recovert phases in a standard compliant way, the same recovery phrase can be used in other wallet apps as well

**Reply by 0x00trust** [reply]
Thank you for your info.
Can you make video to get started.
How to build and deploy app for Android and iOs
Thank you very much

**Comment by 0x00trust** [comment]
Nothing to do
